.rubocop.yml

```diff
@@ -32,3 +32,6 @@ Layout/DotPosition:
 
 Layout/CaseIndentation:
   EnforcedStyle: end
+
+Layout/IndentHash:
+  EnforcedStyle: consistent
```

Comment:
What do you think about ending this file with a newline? You can set your editor to do this automagically, too. [Here's some background info about this](https://thoughtbot.com/blog/no-newline-at-end-of-file).

Reply:
So weird that RuboCop didn't complain to me about this...

Reply:
RuboCop going rogue 😬